bitnami/elasticsearch/templates/master-statefulset.yaml

```diff
@@ -5,6 +5,11 @@ metadata:
   labels: {{- include "elasticsearch.labels" . | nindent 4 }}
     role: master
 spec:
+  updateStrategy:
+    type: {{ .Values.master.updateStrategy.type }}
+    {{- if (eq "Recreate" .Values.master.updateStrategy.type) }}
```

Comment:
According to this, `Recreate` is not a valid `updateStrategy` type for Stateful Sets: https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets

Reply:
good catch. fixed!